var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -80,32 +80,57 @@ def install(self, spec, prefix):
             optflags = {
                 'gcc': ['-O2',
                         '-ffast-math',
-                        '-ffree-form',
-                        '-ffree-line-length-none',
                         '-ftree-vectorize',
                         '-funroll-loops',
                         '-mtune=native'],
                 'intel': ['-O2',
                           '-pc64',
                           '-unroll',
-                          '-heap-arrays 64']
+                          '-xHost']
```

Comment:
>  I am not a fan of the -xHost option.

Me neither, but in this case it seemed to be relevant. I'll dig more into that. To be honest I am not a big fan of hand written Makefiles...